Add support  i2c 16-Channel PWM Driver PCA9685

labastov opened the pull request:
Could you add support  i2c 16-Channel PWM Driver PCA9685

arendst commented:
?? See #1309

ascillato committed:
Update user_config.h

ascillato committed:
Update platformio.ini

gtortone committed:
Merge branch 'development' of https://github.com/arendst/Sonoff-Tasmota into development

gtortone committed:
add new metrics to SDM120 power meter

arendst committed:
Merge pull request #2726 from ascillato/patch-1

KNX: Add control for all Tasmota Outputs.

gtortone committed:
Delete arduino.json

gtortone committed:
Delete c_cpp_properties.json

ascillato committed:
Sync to Tasmota

Sync to Tasmota

arendst committed:
Merge pull request #2727 from gtortone/development

SDM120 power meter - new metrics

arendst committed:
Add user entry DST/STD

5.13.1c
 * Add user entry DST/STD using commands TimeStd and TimeDst
with options like 0,0,3,1,2,120 (#2721)

ascillato committed:
Update Spanish Translation

Update Spanish Translation

ascillato committed:
Update Spanish Translation

Update Spanish Translation

ascillato committed:
Update user_config.h

ascillato committed:
Update from Tasmota

Update from Tasmota

ascillato committed:
Update user_config.h

arendst committed:
Merge pull request #2732 from ascillato/patch-1

Update Spanish Translation

ascillato committed:
Sync with Tasmota

Sync with Tasmota

arendst committed:
Add more user input validation

5.13.1c
 * Add more user input validation to commands TimeStd and
TimeDst

ascillato committed:
Update user_config.h

ascillato committed:
Update from Tasmota

Update from Tasmota

ascillato committed:
Update user_config.h

arendst committed:
v5.14.0a - Sync with release

ascillato committed:
Update platformio.ini

ascillato committed:
v5.14.0a - Sync with release

v5.14.0a - Sync with release

ascillato committed:
Update platformio.ini

ascillato committed:
Update COPYFILES.BAT

ascillato committed:
Update README.md

ascillato committed:
Update README.md

ascillato committed:
Added KNX Communication Enhancement

Added option to enable KNX Communication Enhancement. If activated, will make the device send 3 times the same telegram drastically reducing the telegram lost.

This improves issue #78

ascillato committed:
Upgrade Figures

arendst committed:
Add Eastron SDM630 energy meter

5.14.0a
 * Add Eastron SDM630 energy meter (#2735)

ascillato committed:
Update from Tasmota

Update from Tasmota

ascillato committed:
Update user_config.h

arendst committed:
Update French language file

Update French language file (#2452)

ascillato committed:
Update from Tasmota

Update from Tasmota

arendst committed:
Add KNX communication enhancement

5.14.0a
 * Add KNX communication enhancement (#2742)

arendst committed:
Fix pio compile errors

Fix pio compile errors caused by changed parameters and trict parameter
checking

arendst committed:
Fix pio compile errors

arendst committed:
Fix timer re-trigger after restart

5.14.0a
 * Fix timer re-trigger within one minute after restart (#2744)

ascillato committed:
Update from Tasmota

Update from Tasmota

ascillato committed:
Update platformio.ini

ascillato committed:
Update platformio.ini

ascillato committed:
Update user_config.h

ascillato committed:
Add Energy Information for KNX

Add Energy Information for KNX

ascillato committed:
Add Energy Information for KNX

Add Energy Information for KNX

ascillato committed:
Add Energy Information for KNX

Add Energy Information for KNX

ascillato committed:
Add Energy Information for KNX

Add Energy Information for KNX

ascillato committed:
Update README.md

ascillato committed:
Update

ascillato committed:
Update platformio.ini

crankyoldgit committed:
Allow IR messages with a data payload of zero.

Ref: Issue #2751

Incorrect assumption that if the result of `strtoul()` is zero (0) it is a parse failure. That's incorrect as the "data" payload could actually be 0.

Ref: https://github.com/markszabo/IRremoteESP8266/issues/447#issuecomment-389745404

ascillato committed:
Update README.md

ascillato committed:
Update platformio.ini

arendst committed:
Merge pull request #2750 from ascillato/patch-1

Add Energy Information for KNX

arendst committed:
Merge pull request #2752 from crankyoldgit/patch-1

Allow IR messages with a data payload of zero.

arendst committed:
Admin and some fixes

5.14.0a
 * Add KNX energy data (#2750)
 * Fix display selection of
un-available GPIO options in Module Configuration webpage (#2718)
 * Fix
IRSend not accepting data value of 0 (#2751)

ascillato committed:
Update from Tasmota

Update from Tasmota

arendst committed:
Add compiled feature information

5.14.0a
 * Add compiled feature information to Status 4

ascillato committed:
Upgrade from Tasmota

Upgrade from Tasmota

arendst committed:
Add python firmware server

Python firmware server can provide binary OTA images to Tasmota. See
README for usage. Tested on Linux.

gtortone committed:
fw-server: add ip option

arendst committed:
Merge pull request #2759 from gtortone/development

fw-server: add ip option

ascillato committed:
Update from Tasmota

Update from Tasmota

arendst committed:
Update default lwIP for 2.4.1

Update default lwIP for 2.4.1 (#2745)

ascillato committed:
Update platformio.ini

ascillato committed:
Update from Tasmota

Update from Tasmota

ascillato committed:
Update platformio.ini

gtortone committed:
upgrade of italian language

ascillato committed:
Added Optional Files

Added Optional Files

ascillato committed:
Fix Bug on KNX Menu

Fix Bug on KNX Menu that shows Energy Options while there is no energy sensor selected.

ascillato committed:
Sync with Tasmota

ascillato committed:
Fix Bug on KNX Menu

Fix Bug on KNX Menu that shows Energy Options while there is no energy sensor selected.

ascillato committed:
Updated esp-knx-ip library

Updated esp-knx-ip library with a merged version that allows to select multicast library to use:
* WIFI UDP (for esp lib 2.3.0 and up)
* AsyncUDP (for esp lib 2.4.0 and up)
The selection can be done with a definition key in the file esp-knx-ip.h

ascillato committed:
Fix 'vars' on Rules

The 'vars' was not working from one rule to another because the declaration of the variable was not global and was being lost.

ascillato committed:
Update xdrv_10_rules.ino

arendst committed:
Merge pull request #2769 from ascillato/patch-2

Fix 'vars' on Rules

arendst committed:
Merge pull request #2770 from ascillato/patch-1

 Fix Bug on KNX Menu

arendst committed:
Merge pull request #2766 from gtortone/development

upgrade of italian language

ascillato committed:
Sync to Tasmota

Sync to Tasmota

arendst committed:
Add program to decode some status messages

5.14.0a
* Add feature information to Status 4
* Add tools folder with python script decode-status.py for decoding some
status fields like SetOption and Features
* Add python script fw-server.py in tools folder to create a simple OTA
server (#2759)
* Fix vars on rules (#2769)
* Fix bug in KNX menu (#2770)

znanev committed:
Update Bulgarian translations

Update Bulgarian translations to v5.14.0a

arendst committed:
Merge pull request #2773 from znanev/patch-1

 Update Bulgarian translations

ascillato committed:
Sync to Tasmota

Sync to Tasmota

ascillato committed:
Update stale.yml

arendst committed:
Rules fixes and additions

5.14.0a
* Add rules %mem1% to %mem5% variable names storing data in flash
(#2780)
* Add rules test on %varx% or %memx% (#2780)
* Fix anomalies in rules (#2778)

altelch committed:
Update to HUE Bridge V1 Software Version

arendst committed:
Merge pull request #2788 from altelch/development

Update to HUE Bridge V1 Software Version

ascillato committed:
Sync with Tasmota

Sync with Tasmota

arendst committed:
Add optional token %id% to fulltopic

5.14.0a
* Add optional token %id% substituting the unique MAC address for topic
names by Michael Graf (#2794)

ascillato committed:
Sync with Tasmota

Sync with Tasmota

gtortone committed:
images for wiki

gtortone committed:
remove temporary media directory

arendst committed:
Move GetUsedInModule to support.ino

Move GetUsedInModule to support.ino (#2810)

arendst committed:
Add two more rule sets

5.14.0b
 * Increase rule storage space to 3 rule sets of 512 characters
using commands rule1, rule2 and rule3

arendst committed:
Add rule support for IrReceive and RfReceive

5.14.0b
 * Add rule support for IrReceive and RfReceive (#2758)

arendst committed:
Fix initial rule3 clearence

arendst committed:
Fix rule string comparison

ascillato committed:
Sync to Tasmota

Sync to Tasmota

arendst committed:
Fix Pow R2 and S31 checksum error

Fix Pow R2 and S31 checksum error (#1907)

arendst committed:
Fix sdm120/630 compile error

Fix sdm120/630 compile error when webserver is disabled (#2764)

ascillato committed:
Sync to Tasmota

Sync to Tasmota

arendst committed:
File renames and Add command WebSend

5.14.0b
* Add command WebSend [<host>(:<port>,<user>:<password>)] <command>
(#2821)

arendst committed:
Add command source information

5.14.0b
* Add source information to command execution to be shown with logging
option 3 (#2843)

ascillato committed:
Sync to Tasmota

Sync to Tasmota

arendst committed:
Add Ukranian language

Add Ukranian language

ascillato committed:
Update platformio.ini

ascillato committed:
Sync to Tasmota

Sync to Tasmota

ascillato committed:
Update platformio.ini

ascillato committed:
Update COPYFILES.BAT

arendst committed:
Fix HWRestart on rule event

5.14.0b
* Fix Hardware Watchdog restart when using event command (#2853)

ascillato committed:
Sync to Tasmota

Sync to Tasmota

vivianlink committed:
Update README.md

vivianlink committed:
Update README.md

ascillato committed:
Update README.md

arendst committed:
Merge pull request #2863 from vivianlink/patch-1

fix capitalization error

ascillato committed:
Sync to Tasmota

Sync to Tasmota

arendst committed:
Update language files

Update language files

ascillato committed:
Sync to Tasmota

Sync to Tasmota

ascillato committed:
Add KNX Slots to be used in Rules

ascillato committed:
Update platformio.ini

ascillato committed:
Update user_config.h

ascillato committed:
Added Keys to be used by KNX Console Commands

ascillato committed:
Added KNX Console Commands

* Added Console Commands to send KNX Commands
  usage: KnxTx_Cmnd[slot] command
  where [slot] is any of the 5 slots on the KNX Menu and command is 0 or 1
  example: KnxTx_Cmnd1 0

* Added Console Commands to send KNX Values
  usage: KnxTx_Val[slot] value
  where [slot] is any of the 5 slots on the KNX Menu and value is a number
  example: KnxTx_Val1 35

* Added Slots on the KNX Web Menu to select Group Addess to send data from console commands

* Added Events to trigger rules when received data from KNX
  usage on rules as: event#KnxRx_Val[slot]
  where [slot] is any of the 5 slots on the KNX Menu
  example: rule on event#KnxRx_Val1 do VAR1 %value% endon

* Added Events to trigger rules when received read requests from KNX
  usage on rules as: event#KnxRx_Req[slot]
  where [slot] is any of the 5 slots on the KNX Menu
  example: rule on event#KnxRx_Req1 do KnxTx_Val1 35 endon

* Added Slots on the KNX Web Menu to select Group Addess to receive data to trigger rules

ascillato committed:
Added KNX Console Commands

ascillato committed:
Added Console Commands for TX and RX with KNX

ascillato committed:
Update Optional files

ascillato committed:
Update i18n.h

ascillato committed:
updated optional files

ascillato committed:
Update README.md

ascillato committed:
Update user_config.h

arendst committed:
Merge pull request #2870 from ascillato/patch-1

Add Console Commands for KNX

ascillato committed:
Sync to Tasmota

Sync to Tasmota

ascillato committed:
Add CPU_LOAD Log

ascillato committed:
Add CPU_LOAD Log

arendst committed:
v6.0.0a - Add CRC to Settings

6.0.0a
* Add CRC to Settings making future upgrades more fail-safe
* Remove version 3, 4 and pre 5.2 settings auto-upgrade. See
https://github.com/arendst/Sonoff-Tasmota/wiki/Upgrade#migration-path
* Change default CFG_HOLDER from 0x20161209 to 4617 (=0x1209) - no
impact on default upgrades

arendst committed:
Fix Pzem004T checksum error

6.0.0a
* Fix Pzem004T checksum error

ascillato committed:
Sync to Tasmota

Sync to Tasmota

arendst committed:
Minor fixes

Minor fixes

ascillato committed:
Sync to Tasmota

Sync to Tasmota

arendst committed:
Add SnfBrdg EFM8BB1 fw tools

6.0.0a
* Add support for uploading Sonoff Bridge firmware found in
tools/fw_efm8bb1 folder build by Portisch using Web Gui File Upload
(#2886)
* Add command RfRaw to control Portisch firmware features

andrethomas committed:
Add support for LM75AD I2C Temperature Sensor

Add support for LM75AD I2C Temperature Sensor

ascillato committed:
Sync to Tasmota

Sync to Tasmota

ascillato committed:
Updated Spanish Translation

ascillato committed:
Updated Spanish Translation

Updated Spanish Translation

znanev committed:
Update bg-BG.h

arendst committed:
Merge pull request #2911 from znanev/patch-1

Update Bulgarian translations to v6.0.0a

andrethomas committed:
Update xdrv_26_lm75ad.ino to support C or F

Update xdrv_26_lm75ad.ino to support C or F depending on Settings.flag.temperature_conversion (#define TEMP_CONVERSION 0 or 1) as stored in [SetOption8] Return temperature in (0 = Celsius or 1 = Fahrenheit)

arendst committed:
Merge pull request #2909 from andrethomas/development

Add support for LM75AD I2C Temperature Sensor

arendst committed:
Add I2C temperature sensor LM75AD

6.0.0a
* Add support for I2C temperature sensor LM75AD (#2909)

arendst committed:
Tuning LM75AD sensor driver

arendst committed:
Add RfRaw 255

gtortone committed:
Merge branch 'development' of https://github.com/arendst/Sonoff-Tasmota into development

ascillato committed:
Update Stale Bot

ascillato committed:
Merge branch 'development' into development

ascillato committed:
Revert Changes

ascillato committed:
Revert "Revert Changes"

This reverts commit 1c337ae28eed653b1382d33b6349906d7c191416.

ascillato committed:
Revert "Merge branch 'development' into development"

This reverts commit 3ca69e499b853f59e4a5cd3547ef5612bf394410, reversing
changes made to 7d8495b26325be40ff662b6636d0579a3ed0bbaa.

ascillato committed:
KNX: Fix Bug when reply sensor values

ascillato2 committed:
KNX: Fix Bug when reply sensor values

Solves issue #2734

ascillato committed:
Update formatting of Troubleshooting

ascillato committed:
Update Bug Report Formatting

ascillato committed:
Update Feature Request Formatting

ascillato2 committed:
Update Template Format for Issues

gtortone committed:
Merge branch 'development' of https://github.com/arendst/Sonoff-Tasmota into development

gtortone committed:
update of language it-IT

ascillato2 committed:
Merge pull request #2949 from gtortone/development

update of it-IT language

ascillato committed:
Add Arithmetic commands for VAR

New Arithmetic commands to be used with VARs:

ADD1 to ADD5: Add a value to VAR[x]
Usage: ADD1 15
Result: VAR1 = VAR1 + 15

SUB1 to SUB5: Substract a value from VAR[x]
Usage: SUB1 15
Result: VAR1 = VAR1 - 15

MULT1 to MULT5: Multiply a value to VAR[x]
Usage: MULT1 15
Result: VAR1 = VAR1 * 15

SCALE1 to SCALE5: Scale a value from a low and high limit to another low and high limit and store it in VAR[x] (directly equivalent to MAP arduino command)

Syntax: 
SCALEx value, fromLow, fromHigh, toLow, toHigh

Parameters
value: the number to scale

fromLow: the lower bound of the value’s current range

fromHigh: the upper bound of the value’s current range

toLow: the lower bound of the value’s target range

toHigh: the upper bound of the value’s target range

(omitted values are taken as zero)

Usage: SCALE1 15, 0, 100, 0, 1000
Result: VAR1 = 150

asdadsadsada12 commented:
Hello, some one could help me?
I am trying to integrate this sensor anemometer with sonoff as sensor
https://diyprojects.io/iot-connected-anemometer-wind-vane-esp8266/#.Wx5gnUiFPIU

REgards,

ascillato committed:
Add a Disclaimer to Readme

Add a Disclaimer to Readme due that sonoff devices are being installed by uncertified people

ascillato committed:
KNX: Support for DS18S20 Temperature sensor

ascillato committed:
Update _releasenotes.ino

ascillato committed:
KNX: Support for DS18S20 Temperature sensor

ascillato2 committed:
KNX: Support for DS18S20 Temperature sensor

Drachenkaetzchen committed:
When passing values for multiple WS2812 LEDs at once, updating is very slow because after each LED is set, the strip is updated. The update was so slow that it was visible with the eye. For me, it took approx 0.5 seconds to update an 28 pixel long strip.

This patch suspends updates to the strip while procesing the passed LEDs, enabling the updates afterwards.

Staars committed:
Create xdrv_91_apds9960.ino

First version with the "new" driver model.
The driver is disabled by default to prevent side effects und must be enabled for testing by hand. 
Please uncomment:
// #define USE_APDS9960 

Use GPIO_DSB as interrupt pin as provisional hack in the web interface of tasmota.

Please expect problems (reboot, ...). My personal usage scenario is:
Sonoff -> MQTT -> Node-Red -> MQTT -> Sonoff

Staars committed:
Update xdrv_91_apds9960.ino

changes:
-changed from interrupt mode to polling
-fixed the mqtt-messages
-fixed command "gesture" for the console. Now every argument except 0 and 1 will report the gesture mode
-changed gesture name from "overload" to "long"

killadm committed:
update of zh-CN language

killadm committed:
Update zh-CN.h

Remove >>>>>>

renne committed:
Doxygen configuration file and helper script.

This commit adds a configuration file for the Doxygen documenation generator and a helper script to create/update the source code documentation. Just install Doxygen and GraphViz and run "updateDocs.sh" to create the source code documentation.

Staars committed:
Update xdrv_91_apds9960.ino

new:
-removed NEAR/FAR because of the unreliability
-support for Rules, therefore change of the MQTT-Messages to a number format (clockwise: up,right,down,left then long and none)
Example:
rule1 on Tele-APDS9960#Gesture=1 do ... endon

-minor code changes to shorten things

Staars committed:
Update xdrv_91_apds9960.ino

-add compiler warning (based on input from renne)
-small cosmetic/formatting cleanups

Staars committed:
Update xdrv_91_apds9960.ino

Changed the payload back to string.
I think this is shorter, easier to understand without any documentation and still works with rules.
e.g.
on Tele-APDS9960#Up do command endon

arendst committed:
Merge pull request #3045 from renne/development

Doxygen configuration file and helper script.

arendst committed:
Merge pull request #3041 from killadm/development

update of zh-CN language

arendst committed:
Merge pull request #2972 from ascillato/patch-3

Add a Disclaimer to Readme

arendst committed:
Add debug facilities

Add optional debug facilities and CPU load loop info (#2979)

arendst committed:
Merge pull request #3007 from Drachenkaetzchen/development

Major performace Improvements for WS2812 pixels

arendst committed:
Add hex/decimal receive control for RF and IR

6.0.0a
* Add command SetOption28 to switch between hex or decimal Sonoff Bridge
RF received data format (#3008)
* Add command SetOption29 to switch between hex or decimal IR received
data format

arendst committed:
Merge pull request #2955 from ascillato/patch-2

Add Arithmetic commands for VAR

arendst committed:
Fix xsns_26_lm75ad.ino line endings

Fix xsns_26_lm75ad.ino line endings (#3063)

arendst committed:
Fix rules induced LWT message

Fix rules induced LWT message

arendst committed:
Update decode-status.py

Update decode-status.py

arendst committed:
Add heap and stack debug information

Add heap and stack debug information

arendst committed:
Add rule variables and add minutes

6.0.0a
* Add time in minutes to rule Time#Initialized, Time#set and Time#Minute
(#2669)
* Add rule variables %time% for minutes since midnight, %uptime%,
%sunrise% and %sunset% giving time in minutes (#2669)

arendst committed:
Add option 0 to command Timers

6.0.0a
* Add option 0 to command Timers disarming all timers (#2962)

arendst committed:
Add increment and decrement counter

6.0.0a
* Add increment and decrement value to command Counter (#2838)

arendst committed:
Merge pull request #3051 from Staars/development

APDS9960 (my second try)

ascillato committed:
Change link of SonWeb to TasmoAdmin

arendst committed:
Add support for APDS9960 proximity sensor

6.0.0a
* Add support for APDS9960 proximity sensor (#3051)

arendst committed:
Merge pull request #3072 from ascillato/patch-1

Change link of SonWeb to TasmoAdmin

arendst committed:
Renamed file and updated APDS9960 sensor

Renamed file and updated APDS9960 sensor

Jason2866 committed:
Update de-DE.h

arendst committed:
Merge pull request #3079 from Jason2866/patch-3

Update EFM Pogrammer Part in de-DE.h

StofLE committed:
Update de-DE.h

Update KNX Part in de-DE.h

arendst committed:
Merge pull request #3081 from StofLE/development

Update de-DE.h

arendst committed:
Fix possible wifi connection problem

Fix possible wifi connection problem (#1366)

arendst committed:
v6.0.0b - Support up to 8 switches

6.0.0b
* Change number of switches from 4 to 8 (#2885, #3086)

arendst committed:
Add experimental (untested) TM1638 support

Add experimental (untested) TM1638 switch support (#2226)

ascillato committed:
KNX: Command KNXTX_VALx now supports value with decimals

arendst committed:
Merge pull request #3088 from ascillato/patch-1

Add decimal values support for Command KNXTX_VALx

ascillato committed:
Added decimal values support for commands ADD, SUB, MULT and SCALE

arendst committed:
Merge pull request #3089 from ascillato/patch-2

Add decimal values support for commands ADD, SUB, MULT and SCALE

arendst committed:
v6.0.0b

arendst committed:
Add enforce Hass discovery as light group

Add command SetOption30 to enforce Hass discovery as light group (#1784)

arendst committed:
Add support for Sonoff S26 Smart Socket

Add support for Sonoff S26 Smart Socket (#2808)

arendst committed:
Allow increase in friendlynames

Allow increase in friendlynames (#3098)

arendst committed:
Add some delay to TM1838 comms

Add some delay to TM1838 comms (#2226)

arendst committed:
Add some delay to TM1638 - better

Add some delay to TM1638 (#2226)

arendst committed:
Initial Sonoff iFan02 support

Initial Sonoff iFan02 support (#2839)

arendst committed:
Add restriction

Add restriction

arendst committed:
Quick fix to re-instate iFan02 RC

Quick fix to re-instate iFan02 RC

arendst committed:
Add Sonoff iFan02 support

Add support for Sonoff iFan02 as module 44 introducing command FanSpeed
0..3 (#2839)

arendst committed:
Fix iFan02 friendlyname

Fix iFan02 friendlyname

arendst committed:
Reserve space for MCP23017

Reserve space for MCP23017 (#3107)

arendst committed:
Add support for BlitzWolf BW-SHP2

Add support for BlitzWolf BW-SHP2 (and Homecube, Gosund SP1) Energy
Monitoring Smart Socket (#2223)

arendst committed:
Add migration instructions

Add migration instructions

arendst committed:
v6.0.0c - Add spare SetOptions

6.0.0c
* Add support for bitflags SetOption50 .. SetOption81 (#3118)

arendst committed:
Rewrite BME680 driver

Rewrite BME680 driver now using latest Bosch BME680 library (#2969)

arendst committed:
Updated to v6.1.0a

Updated to v6.1.0a

matias-b committed:
Merge pull request #1 from arendst/development

Updated to v6.1.0a

matias-b committed:
Fix es-AR typos

arendst committed:
Merge pull request #3143 from matias-b/language-fix

es-AR language fix

arendst committed:
Fixed minimal

enabled HTTPCommand for minimal

arendst committed:
Add wifi/mqtt status blinkyblinky

6.1.0a
 * Add wifi and mqtt status led blinkyblinky to be disabled by SetOption31 1. Does not work when LedPower is On (deliberate) (#871, #2230, #3114, #3155)
 * Remove not needed functionality from Sonoff-minimal to save space

arendst committed:
Change SHT1x sensor detection

Change SHT1x sensor initialization from pre-teleperiod to once during restart to fix I2C interference

arendst committed:
Update TSL2561 library

Remove TSL2561 debug message and update library (#2415)

arendst committed:
Change DS18B20 driver

Change DS18B20 driver to provide better instant results

arendst committed:
Change DS18x20 driver

Change DS18x20 driver to provide better instant results (#3169)

arendst committed:
Change DHT driver

Change DHT driver to provide better instant results and add decimals to DHT11 (#3164)

arendst committed:
Fix DHT driver

Fix DHT driver mixing values for different sensors (#1797)

arendst committed:
Change SHT1x driver

Change SHT1x driver to provide better instant results

arendst committed:
Add read sensor retry

Add read sensor retry to DS18B20, DS18x20, DHT, SHT1X and HTU21

arendst committed:
Update sensor drivers

Update sensor drivers to provide instant results

arendst committed:
Updated bmp driver

Updated bmp driver

arendst committed:
Update sensor drivers

Update sensor drivers

arendst committed:
Add profile info to drivers

Update release notes and add profile info to some drivers

arendst committed:
Fix invalid response

Fix invalid response using more than 4 switches and domoticz

arendst committed:
Add TM1638 switch support

Add TM1638 switch support (#2226)

arendst committed:
Fix TM1638 compile error

Fix TM1638 compile error (#3212)

arendst committed:
v6.1.0b - WPS optional

6.1.0b
 * Add user selection of WPS as define USE_WPS in user_config.h in preparation for core v2.4.2 (#3221)
 * Change default Wifi config option from WPS to Wifi Manager if WPS is disabled (or Wifi Smartconfig if webserver is disabled)
 * Remove WPS from sonoff-minimal saving 33k code space
 * Revert wifi changes implemented in v6.0.0a due to possible bad initial wifi connections

arendst committed:
v6.1.1a - Sync with v6.1.1

Sync with v6.1.1

ascillato2 commented:
Hi,

This pull request is trying to merge the Dev with Master of this Repository. Is not adding any code.

Seems that this PR was done by mistake trying to make a feature request.

Closing.
